topics/climate/tutorials/climate-101/tutorial.md

```diff
@@ -397,6 +405,8 @@ When it comes to future climate, we usually need to make some assumptions (such
 
 # Conclusion
 
+This tutorial taught you some of the basics of climate data analysis, explaining the difference between weather and climate as well as the existance of Essential Climate Variables to help you choosing variables that can help you in your analysis. In future Galaxy tutorials, we will go into details on where to get Climate data and how to make statistical analysis to get an insight on the climate change.
```

Comment:
this is now semi-redundant with the sentence below.

Reply:
Very redundant. I forgot we had already a conclusion! I removed what I just added...